Assets/Scripts/Characters/Human/TriggerColliderWeapon.cs

```diff
@@ -9,360 +9,238 @@ public class TriggerColliderWeapon : MonoBehaviour
 {
     public Equipment Equipment { get; set; }
 
-    public bool active_me;
-    public GameObject currentCamera;
+    public bool IsActive;
+    public IN_GAME_MAIN_CAMERA currentCamera;
     public ArrayList currentHits = new ArrayList();
     public ArrayList currentHitsII = new ArrayList();
     public AudioSource meatDie;
+    public Hero hero;
     public int myTeam = 1;
     public float scoreMulti = 1f;
+    public Rigidbody body;
 
-    private bool checkIfBehind(GameObject titan)
+    private FengGameManagerMKII manager;
+
+    private bool IsBehind(GameObject titan)
     {
         if (titan.transform.Find("Amarture/Core/Controller_Body/hip/spine/chest/neck/head") != null)
         {
             Transform transform = titan.transform.Find("Amarture/Core/Controller_Body/hip/spine/chest/neck/head");
-            Vector3 to = base.transform.position - transform.transform.position;
+            Vector3 to = transform.position - transform.transform.position;
             return (Vector3.Angle(-transform.transform.forward, to) < 70f);
         }
         else if (titan.transform.Find("BodyPivot/HeadPos") != null)// dummy titan
         {
             Transform transform = titan.transform.Find("BodyPivot/HeadPos");
-            Vector3 to = base.transform.position - transform.transform.position;
+            Vector3 to = transform.position - transform.transform.position;
             return (Vector3.Angle(-transform.transform.forward, to) < 70f);
         }
         return false;
     }
 
     public void DummyNapeHit(DummyTitan titan)
```

Comment:
This method is unused. I think this should be somewhere within the case "titanneck"near line 109

Reply:
Ah, true, thought it was used somewhere else, should be fixed in the next commit.